client/my-sites/controller.js

```diff
@@ -375,6 +378,14 @@ export function siteSelection( context, next ) {
 
 	const siteId = getSiteId( getState(), siteFragment );
 	if ( siteId ) {
+		const isAtomicSite = isSiteAutomatedTransfer( getState(), siteId );
+		const calypsoify = isAtomicSite && abtest( 'calypsoifyPlugins' ) === 'pointToWPAdmin';
+
+		if ( calypsoify && /^\/plugins/.test( basePath ) ) {
+			const pluginLink = getSiteAdminUrl( getState(), siteId ) + 'plugin-install.php?calypsoify=1';
+			return window.location.replace( pluginLink );
```

Comment:
Let's check for window before assuming it's available. We do have a few SSR'd pieces, though I'm not sure we'd hit it here